tests/sqllogictests/suites/base/03_common/03_0000_select_aliases

```diff
@@ -64,3 +64,10 @@ SELECT b.number FROM (SELECT * FROM numbers(3) AS b ORDER BY a.number) AS a
 statement error 1065
 SELECT b.number FROM (SELECT * FROM numbers(3) AS b) AS a ORDER BY b.number
 
+query I
+select number * 2 as a from numbers(3) where (a + 1) % 3 = 0
+----
+2 
+
+statement error
+select number, number * 2 as number from numbers(3) where (number + 1) % 3 = 0
```

Comment:
select number * 2 as number from numbers(3) where (number + 1) % 3 = 0

Should this throw error?

Reply:
This will not throw an error now. What do you think is the best?